editor/plugins/node_3d_editor_gizmos.cpp

```diff
@@ -3453,7 +3453,7 @@ void ReflectionProbeGizmoPlugin::set_handle(const EditorNode3DGizmo *p_gizmo, in
 
 		Vector3 ra, rb;
 		Geometry3D::get_closest_points_between_segments(Vector3(), axis * 16384, sg[0], sg[1], ra, rb);
-		float d = ra[p_id];
+		float d = ra[p_id] * 2.0;
```

Comment:
This looks suspicious to me, but I am unfamiliar with this area of code

Reply:
Seems to work correctly.